src/clr-angular/wizard/wizard.module.ts

```diff
@@ -1,5 +1,5 @@
-/**
- * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
```

Comment:
Just wondering if we should put an explicit link?

Reply:
Link to the LICENSE.txt file on github? Sure I can add that.